src/templates/Front/liveTeasers.js

```diff
@@ -34,8 +35,62 @@ const createLiveTeasers = ({
   DiscussionLink = DefaultLink,
   t,
   withFeedData = withData,
-  withDiscussionsData = withData
+  withDiscussionsData = withData,
+  withMyMagazineData = withData,
+  ActionBar,
+  showMyMagazine = true
 }) => {
+  const MyMagazineWithData = withMyMagazineData(
+    ({ attributes, data, url, label }) => {
+      return (
+        <Loader
+          error={data.error}
+          loading={data.loading}
+          style={{ minHeight: 600 }}
```

Comment:
I'd recommend to move the `600` to a shared variable to ensure that the `Loader` and `LazyLoad` placeholder always have the same size.